### PR TITLE
[Paddle]Add chapter_convolutional-modern-alexnet

### PR DESCRIPTION
**Pull request by tngt**

### CODE REVIEW

**Comment by w5688414** [comment]
@AnirudhDagar @cheungdaven chapter convolutional modern alexnet is ready, can you review this chapter? Thanks

**Comment by tngt** [comment]
> @tngt Could you resolve the conflicts?
> 
> The performance of paddlepaddle in the batch_norm notebook is much worse than other libraries, could you take a look?
> 
> [batch_norm](http://preview.d2l.ai/d2l-zh/PR-1154/chapter_convolutional-modern/batch-norm.html)
> 
> The accuracy of paddle: loss 0.252, train acc 0.908, test acc 0.782 27811.7 examples/sec on CUDAPlace(0)
> 
> Pytorch: loss 0.269, train acc 0.901, test acc 0.876 60353.3 examples/sec on cuda:0

We fixed it, could you review it again?

**Comment by tngt** [comment]
@cheungdaven  could you review it again? thanks!!!
